triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/simulation/TriggerMultiProcessSimulation.scala

```diff
@@ -25,13 +24,6 @@ abstract class TriggerMultiProcessSimulation extends AsyncWordSpec with Abstract
   protected implicit lazy val simulationConfig: TriggerSimulationConfig =
     TriggerSimulationConfig()
 
-  protected implicit lazy val simulation: ActorSystem[Message] =
-    ActorSystem(triggerMultiProcessSimulationWithTimeout, "simulation")
-
-  override implicit lazy val materializer: Materializer = Materializer(simulation)
-
-  override implicit lazy val executionContext: ExecutionContext = materializer.executionContext
```

Comment:
These implicits no longer need to be overridden by this code (thus side stepping cyclic dependency/init issues)